easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.1-foss-2016a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,24 @@
+name = 'SuiteSparse'
+version = '4.5.1'
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+metis = 'METIS'
+metis_ver = '5.1.0'
+versionsuffix = '-%s-%s' % (metis, metis_ver)
+dependencies = [(metis, metis_ver)]
+
+preconfigopts = 'LAPACK="$LIBLAPACK" ' 
+prebuildopts = 'LAPACK="$LIBLAPACK" ' 
+preinstallopts = 'LAPACK="$LIBLAPACK" ' 
```

Comment:
@besserox @hajgato @boegel I see that the problem will be solved. but just in case I leave here part of my log when I tried to install SuiteSparse without setting LAPACK variable:

gcc -fPIC -funroll-loops -O3 -march=native 
-I/home/CLUSTER/uclv.jrbosch/.local/easybuild/software/OpenBLAS/0.2.15-GCC-4.9.3-2.25-LAPACK-3.6.0/include 
-I/home/CLUSTER/uclv.jrbos$
gcc -fPIC -funroll-loops -O3 -march=native 
-I/home/CLUSTER/uclv.jrbosch/.local/easybuild/software/OpenBLAS/0.2.15-GCC-4.9.3-2.25-LAPACK-3.6.0/include 
-I/home/CLUSTER/uclv.jrbos$
gcc -fPIC -funroll-loops -O3 -march=native 
-I/home/CLUSTER/uclv.jrbosch/.local/easybuild/software/OpenBLAS/0.2.15-GCC-4.9.3-2.25-LAPACK-3.6.0/include 
-I/home/CLUSTER/uclv.jrbos$
./cholmod_demo < Matrix/bcsstk01.tri
./cholmod_demo: symbol lookup error: /usr/lib/liblapack.so.3: undefined symbol: gotoblas
Makefile:31: recipe for target 'all' failed
make[2]: *** [all] Error 127
make[2]: Leaving directory 
'/home/CLUSTER/uclv.jrbosch/.local/easybuild/build/SuiteSparse/4.5.2/foss-2016a-METIS-5.1.0/SuiteSparse/CHOLMOD/Demo'
Makefile:14: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory 
'/home/CLUSTER/uclv.jrbosch/.local/easybuild/build/SuiteSparse/4.5.2/foss-2016a-METIS-5.1.0/SuiteSparse/CHOLMOD'
Makefile:14: recipe for target 'go' failed
make: *** [go] Error 2
 (at 
easybuild/software/EasyBuild/2.7.0/lib/python2.7/site-packages/easybuild_framework-2.7.0-py2.7.egg/easybuild/tools/run.
py:397 in parse_cmd_output)
== 2016-05-11 23:56:54,700 easyblock.py:2296 WARNING build failed (first 300 chars): cmd 
"MY_METIS_LIB="$EBROOTMETIS/lib/libmetis.so"  make -j 1 " exited with exitcode 2 and ou$
Using pre-installed METIS 5.1.0 library at  
[/home/CLUSTER/uclv.jrbosch/.local/easybuild/software/METIS/5.1.0-foss-2016a/lib/libmetis.so]
( cd SuiteSparse_config && make )
make[1]: Entering directory '/h
== 2016-05-11 23:56:54,700 easyblock.py:267 INFO Closing log for application name SuiteSparse version 4.5.2